tools/cli/app_test.go

```diff
@@ -128,26 +117,22 @@ func (s *cliAppSuite) TestAppCommands() {
 
 func (s *cliAppSuite) TestDomainRegister_LocalDomain() {
 	s.serverFrontendClient.EXPECT().RegisterDomain(gomock.Any(), gomock.Any()).Return(nil)
-	errorCode := s.RunErrorExitCode([]string{"", "--do", domainName, "domain", "register", "--global_domain", "false"})
-	s.Equal(0, errorCode)
+	s.NoError(s.app.Run([]string{"", "--do", domainName, "domain", "register", "--global_domain", "false"}))
 }
 
 func (s *cliAppSuite) TestDomainRegister_GlobalDomain() {
 	s.serverFrontendClient.EXPECT().RegisterDomain(gomock.Any(), gomock.Any()).Return(nil)
-	errorCode := s.RunErrorExitCode([]string{"", "--do", domainName, "domain", "register", "--global_domain", "true"})
-	s.Equal(0, errorCode)
+	s.NoError(s.app.Run([]string{"", "--do", domainName, "domain", "register", "--global_domain", "true"}))
 }
 
 func (s *cliAppSuite) TestDomainRegister_DomainExist() {
 	s.serverFrontendClient.EXPECT().RegisterDomain(gomock.Any(), gomock.Any()).Return(&types.DomainAlreadyExistsError{})
-	errorCode := s.RunErrorExitCode([]string{"", "--do", domainName, "domain", "register", "--global_domain", "true"})
-	s.Equal(1, errorCode)
+	s.Error(s.app.Run([]string{"", "--do", domainName, "domain", "register", "--global_domain", "true"}))
 }
 
 func (s *cliAppSuite) TestDomainRegister_Failed() {
 	s.serverFrontendClient.EXPECT().RegisterDomain(gomock.Any(), gomock.Any()).Return(&types.BadRequestError{"fake error"})
-	errorCode := s.RunErrorExitCode([]string{"", "--do", domainName, "domain", "register", "--global_domain", "true"})
-	s.Equal(1, errorCode)
+	s.Error(s.app.Run(([]string{"", "--do", domainName, "domain", "register", "--global_domain", "true"})))
```

Comment:
this may be more effort than is desired at the moment and that's completely fine if so, but:

any chance I can encourage you to switch these to something like this?
```go
s.ErrorContains(
  s.app.Run(...),
  "domain registration failed",
)
```

Reply:
I'll at least fix this file

Reply:
I'll take back my word. It's a lot more effort than a pure package upgrade. Some commands doesn't error wrapping but simply return the original error. 
For example, in this case, it's return "fake error" which is pretty bad. Fixing it is going to be another PR in the future. At least, I've put a TODO here.